### ISSUE TITLE
Add option for custom command launch

### ISSUE THREAD

**Issue by the-sushi** [nbedos/termtosvg#83]
Instead of only offering the default shell, it would be useful to have a flag that allows for the launch of a custom program without having to go through the shell first.

For example:

Current behavior:
```
$ termtosvg myfile.svg
Recording started, enter "exit" command or Control-D to end
$ mycommand
 # interact with mycommand
$ exit
Recording ended, SVG animation is myfile.svg
```

Requested behavior:
```
$ termtosvg myfile.svg -c mycommand
 # interact with mycommand
Recording ended, SVG animation is myfile.svg
```

**Comment by nbedos** [comment]
I forgot about this, thanks for reminding me. Yes, that would help for #20 as described [here](https://github.com/nbedos/termtosvg/issues/20#issuecomment-402803532) and also for #11.

**Comment by nbedos** [comment]
There is an open pull request for this feature here: #84 
Suggestions, reviews and testing welcome.

**Comment by nbedos** [comment]
Feature included in [version 0.7.0](https://github.com/nbedos/termtosvg/releases/tag/0.7.0).
